src/views/OutputComponent.tsx

```diff
@@ -11,7 +11,7 @@ const CharGroupComponent = ({group}: {group: Char[]}) => {
     return (
         <span
             className="char-group"
-            style={css.charGroup(attributes)}>
+            style={attributes ? css.charGroup(attributes) : null}>
```

Comment:
How about `attributes && css.charGroup(attributes)` or this will not work here?

Reply:
`attributes && css.charGroup(attributes)` is work.